docs/deployjiva.md

```diff
@@ -5,64 +5,63 @@ sidebar_label: Provisioning Jiva
 ---
 ------
 
-OpenEBS can be used to create Storage Pool on a host disk or an externally mounted disk. This Storage Pool can be used to create Jiva volume which can be utilized to run applications. By default, Jiva volume will be deployed on host path. If you are using an external disk, see [storage pool](/docs/next/setupstoragepools.html#configuring-a-storage-pool-on-openebs) for more details about creating a storage pool with an external disk. 
+OpenEBS can be used to create Storage Pool on a host disk or an externally mounted disk. This Storage Pool can be used to create Jiva volume which can be utilized to run applications. By default, Jiva volume will be deployed on the host path. If you are using an external disk, see [Storage Pools](/docs/next/setupstoragepools.html#configuring-a-storage-pool-on-openebs) section for more details about creating a storage pool with external disk(s).
```

Comment:
This Storage Pool can be used to create Jiva volume = You can use this Storage Pool to create a Jiva volume...